otherlibs/systhreads/st_stubs.c

```diff
@@ -233,13 +233,22 @@ static void caml_thread_enter_blocking_section(void)
 
 static void caml_thread_leave_blocking_section(void)
 {
+#ifdef _WIN32
```

Comment:
There are no other uses of `#ifdef _WIN32` in this file (other than the initial include on L51). Perhaps it would be better to follow the existing convention and factor out the `caml_thread_leave_blocking_section` function itself into the `st_posix.h` and `st_win32.h` files?

Reply:
Indeed! My concern is that either it introduces two hooks, basically called `do_windows_only_thing_before_lock_acquire` and `do_windows_only_thing_after_lock_acquire` or the entire function logic is duplicated in both headers 🙂